.github/workflows/doxygen.yml

```diff
@@ -0,0 +1,30 @@
+name: Doxygen
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Should be `main`

Reply:
This should hopefully get resolved simply with rebasing or merging the PR with latest

Reply:
oh wait, nevermind, new file 😄

Reply:
Yep, gotta change this to main now

Reply:
Changed this to main